integreat_cms/static/src/css/style.scss

```diff
@@ -53,6 +53,7 @@ header {
 nav#primary-navigation {
   #menu {
     max-height: calc(100vh - 50px);
+    padding-bottom: 3rem;
```

Comment:
AFAICT `2.5 rem` should be sufficient? But it works as is. It could be a little more space efficient though :D

Reply:
I changed it to 2.5rem now. However with this change 10px get cut off at the bottom. Can we maybe settle at 2.75rem? :D

Reply:
I agree with @svenseeberg, `2.5rem` should be the perfect value. The version string has a line height of `1.5rem` and a padding of `0.5rem`, so `1.5 + 0.5 + 0.5 = 2.5`... I cannot reproduce that anything is cut off or 10px are missing somewhere...

Reply:
Oh, I'm sorry, now I can indeed reproduce the small overlap of 6px :see_no_evil: 
I think this is due to the incorrect calculation of the max height... the logo has a height of 56px, but only 50px are deducted from 100vh for the menu...